lib/env/config.js

```diff
@@ -60,6 +62,13 @@ module.exports = {
       return customConfigContent;
     }
     const configPath = getConfigPath();
-    return Promise.resolve(require(configPath)); // eslint-disable-line
+    try {
+      return Promise.resolve(moduleLoader.require(configPath));
+    } catch (e) {
+      if (e.code === 'ERR_REQUIRE_ESM') {
```

Comment:
@nathan-knight 
I observe instability here (potentially depending on the Node version used ???)
This code used to work for me, I used it some weeks ago but now for the same usage I no longer have this error code thrown by the Node's require so it seems unstable.

Reply:
This error only occurs on newer versions of Node that are configured to use ES modules